src/vivarium/framework/event.py

```diff
@@ -30,6 +30,7 @@
 
 import pandas as pd
 
+from ..manager import Manager
```

Comment:
I did not know you could do this `..` on imports.

Reply:
PyCharm did that for me. I'm not sure I like it now that I look at it carefully...